README.md

```diff
@@ -101,101 +101,6 @@ which will build all the necessary files into the `webapp` directory. The versio
 will not appear in Settings without using the dist script. You can then mount the
 `webapp` directory on your webserver to actually serve up the app, which is entirely static content.
 
-config.json
```

Comment:
This section is hard referenced by the app as https://github.com/vector-im/riot-web#configjson

We should fix the app links and maintain this section saying "See docs/config.md"

Reply:
Okay, I'll create a React SDK PR to update the app link.

I assume you want to revive this section with a pointer to the new file just for old Riot versions with the current link? If so, could we eventually remove it after some time has passed?

Reply:
I restored the section, but lower down in the file so that the different ways of running Riot show up first.

Reply:
yup, just want to maintain the link for backwards compatibility reasons. I think our uptake is about 8 weeks, so in about 8 weeks we can probably remove it (or just whenever someone flails at the readme again)